### PR TITLE
feat: added FAQ in footer

### LINKED ISSUE #4539: FAQ in footer
### Is your feature request related to a problem? Please describe

Adding the **FAQ** section in the footer will help users get a basic idea regarding CV and how they can use it. Those all are the most asked questions as well as common ones and bringing that here adds value.

Screenshot from 2024-01-08 12-19-11

### Describe the solution you'd like

As we have an **FAQ** section ready to use(In the User Manual) it is just a matter of adding a tag and hyperlink in the footer accordingly.

Screenshot from 2024-01-08 12-11-15

### Describe alternatives you've considered

Instead of using a user manual FAQ for navigation, we can even build a dedicated FAQ by considering the forum section question or brainstorming the questions 

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by igennova**
Fixes #4539 

#### Describe the changes you have made in this PR -
Added FAQ in footer .

### Screenshots of the changes (If any) -
image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -38,6 +38,7 @@
             <a href="/contribute"><h6><%= t("layout.footer.link_to_contribute") %></h6></a>
             <a href="/teachers"><h6><%= t("layout.link_to_teachers") %></h6></a>
             <a href="/about"><h6><%= t("layout.link_to_about") %></h6></a>
+            <a href="https://docs.circuitverse.org/#/chapter8/2cvfaq"><h6><%=t("layout.link_to_faq") %></h6></a>
```
Comment (smritigarg) [inline app/views/layouts/_footer.html.erb:41]:
```suggestion
            <a href="https://docs.circuitverse.org/#/chapter8/2cvfaq"><h6><%= t("layout.link_to_faq") %></h6></a>
```

**Comment by smritigarg** [comment]
Thanks @igennova Change looks good to me, just a minor styling suggestion here :) which is a non blocker. Approving the same 🚀

**Review by smritigarg — APPROVED** [review]

**Comment by igennova** [comment]
> Thanks @igennova Change looks good to me, just a minor styling suggestion here :) which is a non blocker. Approving the same 🚀 @tachyons I think you will need to merge this one 🤔

Thanks

**Review by tanmoysrt — APPROVED** [review]
Looks good :+1:

**Pull request merged**